examples/mithril-kitchen-sink/README.md

```diff
@@ -0,0 +1,3 @@
+# Storybook Demo
+
+This is a demo app to test Mithril integration with Storybook. Run `npm install` to sync Storybook module with the source code and run `npm run storybook` to start the Storybook.
```

Comment:
why "npm install", doesn't it bootstrapped with yarn workspace?

Reply:
I just copied it from the vue-kitchen-sink. I'm glad to tell me a good readme since I'm not good at English.